mod_config/forms.py

```diff
@@ -7,44 +8,53 @@
 from mod_config.models import Service, Notification
 
 
-def is_python(file_name):
+class FileType(Enum):
+    PYTHONFILE = 1
+    CONTAINER = 2
+
+
+def is_python_or_container(file_name):
     # Check if it ends on .py
-    is_py = re.compile("^[^/\\\]*\.py$")
-    if not is_py.match(file_name):
-        raise ValidationError('Provided file is not a python (.py) file!')
+    is_py = re.compile("^[^/\\\]*\.py$").match(file_name)   
```

Comment:
You might want to use a `raw` string instead so that it's no problem for the PEP check :)